sirepo/server.py

```diff
@@ -294,12 +294,24 @@ def app_find_by_name(simulation_type, application_mode, simulation_name):
 def app_import_file(simulation_type):
     f = flask.request.files['file']
     try:
-        srwblParam, appParam, el, pp = sirepo.sirepo_parser.sirepo_parser(f.read())
-        args = sirepo.sirepo_parser.list2dict(srwblParam)
-        v = sirepo.sirepo_parser.Struct(**args)
-        args = sirepo.sirepo_parser.list2dict(appParam)
-        app = sirepo.sirepo_parser.Struct(**args)
-        data = sirepo.sirepo_parser.parsed_dict(v, app, el, pp)
+        from werkzeug import secure_filename
+        filename = secure_filename(f.filename)
+        imported_file_path = os.path.join('./', filename)
+        try:
+            f.save(imported_file_path)
+        except:
+            raise Exception('File <{}> cannot be saved.'.format(filename))
+
+        # Here we provide path to mirror_1d.dat .dat file and use it for each element requiring profile as placeholder:
+        lib_dir = str(_simulation_lib_dir('srw'))
```

Comment:
Do not hardwire strings. This should either be passed in.

This brings up the point of the file name. This should be in an "importer" (just like template) directory and it should be named "srw_python.py", because it imports python only.